app/assets/javascripts/actions/wikidata_actions.js

```diff
@@ -2,28 +2,27 @@ import { chunk, map, join, filter } from 'lodash-es';
 import * as types from '../constants';
 import logErrorMessage from '../utils/log_error_message';
 import CourseUtils from '../utils/course_utils';
+import request from '../utils/request';
+
 
 const wikidataApiBase = 'https://www.wikidata.org/w/api.php?action=wbgetentities&format=json&origin=*';
 
-const fetchWikidataLabelsPromise = (qNumbers) => {
+const fetchWikidataLabelsPromise = async (qNumbers) => {
```

Comment:
This appears to be broken. The `body` params are not being included.

I wrote a test that uses this (and breaks with your change): https://github.com/WikiEducationFoundation/WikiEduDashboard/commit/b9866a6071a842e10574c706364545ae9c064ba7

Reply:
Should be fixed now